.gitignore

```diff
@@ -1,6 +1,6 @@
-
 *.class
 *.log
+libs/
```

Comment:
We can't ignore libs because the only lib that it contains is `dokkaJavaApi` which is created by our subproject and is not published anywhere. It contains some bypasses to dokka plugin API and few other methods that allow us to actually run dokka.

Reply:
Aaaah nevermind, you fixed that ;)